### ISSUE TITLE
Undefined id property results in Fatal error

### ISSUE THREAD

**Issue by errant-info** [Bannerets/keybase-export#16]
This chat extends back several years.  I'm unable to create any export due to the following error.  
```
./bin/keybase-export config.json                                     [2023-03-31 19:52:36]

Initializing...
Getting chat list...
Total chats: 25
Started loading messages: tranny_mayhem#general
Received new chunk (304 msgs): tranny_mayhem#general
[Fatal error] TypeError: Cannot read property 'id' of undefined
    at convertMessage (/Users/ndawsonfresh/dev/keybase-export/dist/index.js:97:85)
    at /Users/ndawsonfresh/dev/keybase-export/dist/index.js:268:33
    at Array.map (<anonymous>)
    at /Users/ndawsonfresh/dev/keybase-export/dist/index.js:268:22
    at Generator.next (<anonymous>)
    at fulfilled (/Users/ndawsonfresh/dev/keybase-export/dist/index.js:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
```

Perhaps just print a dump of the object and skip the item? 

Can confirm I'm able to successfully export other chats.

**Comment by eilvelia** [comment]
It's weird that keybase can send undefined messages, I've never encountered this issue. I tried to fix it in b2a9501022ee516a8377aa7b3abad51dc86d73da, could you confirm it works now?

**Comment by errant-info** [comment]
Confirmed, works now - thanks!
